modules/logging/src/main/scala/com/evolutiongaming/skafka/producer/Log.scala

```diff
@@ -0,0 +1,20 @@
+package com.evolutiongaming.skafka.producer
+
+import cats.effect.Async
+import com.evolutiongaming.safeakka.actor.ActorLog
+
+trait Log[F[_]] {
```

Comment:
FYI there is one in `cats-helper`

Reply:
not sure if applicable here, but could be useful: https://github.com/ChristopherDavenport/log4cats and supporting lib for metrics: https://github.com/ChristopherDavenport/epimetheus-log4cats

Reply:
This is not a generic logger but an adapter around `ActorLog`. `cats-helper` and `log4cats` are both wrapped around Slf4j which doesn't seem to be the point here.

Reply:
As discussed, dropped `safe-actor` dependency. Used `cats-helper`'s log instead.